pkg/agent/proxy/proxier.go

```diff
@@ -528,7 +531,7 @@ func (p *proxier) installServices() {
 			// group whose Endpoints are local.
 			nGroupID := groupID
 			if svcInfo.NodeLocalExternal() {
-				nGroupID, _ = p.groupCounter.Get(svcPortName, true)
+				nGroupID = p.groupCounter.AllocateNotExist(svcPortName, true)
```

Comment:
I think you should update the call in `GetServiceFlowKeys`, otherwise group 0 would be returned?

Reply:
Thanks, updated. `Get` is only used for deletion now.